### PR TITLE
rustup: new package (1.25.1)

### PR DESCRIPTION
**Pull request by iamwacko**
With this `synth` and `oniongen-rs` won't have to have those weird prepare statements

### CODE REVIEW

File: rustup/lure.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name='rustup'
+version='1.25.1'
+release='1'
+desc='Rust toolchain manager'
+homepage='https://rustup.rs/'
+maintainer='s e <iamawacko@protonmail.com>'
+architectures=('amd64' 'aarch64' 'arm7' '386')
+license=('MIT', 'Apache-2.0')
```
[the remark this answers is not in the document]
Comment (Arsen6331) [reply rustup/lure.sh:8]: Ah, bash arrays don't have commas. It interprets `'MIT',` as a string `"MIT,"`

**Comment by Arsen6331** [comment]
Also, you don't need to use `wget` for this, you can just add the links as sources.

**Comment by iamwacko** [comment]
> Also, you don't need to use `wget` for this, you can just add the links as sources.

I was getting some errors when I tried to do that, but updating lure seems to have resolved that

**Comment by Arsen6331** [comment]
Thanks, merging

**Pull request merged**